src/library/crate/cratestorage.h

```diff
@@ -287,6 +290,8 @@ class CrateStorage: public SqlStorage {
             CrateId crateId) const;
     CrateTrackSelectResult selectTrackCratesSorted(
             TrackId trackId) const;
+    CrateTrackSelectResult selectTracksSortedByCrateNameLike(
```

Comment:
"selectCrateTracks..." to follow the naming style of its siblings. The result contains (crateId, trackId) tuples and not only trackIds.

Now that we added a new function we could think about how to improve the naming of all related functions. But finally it it should be consistent.